traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -345,10 +345,10 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		{Version: api.Version{Major: 5, Minor: 0}, Method: http.MethodGet, Path: `system/info/?$`, Handler: systeminfo.Get, RequiredPrivLevel: auth.PrivLevelReadOnly, RequiredPermissions: nil, Authenticated: Authenticated, Middlewares: nil, ID: 42104747531},
 
 		//Type: CRUD
```

Comment:
nit: Since this is the first time for creating this functions (in API v5), I don't think you need to specify `V5` in the functions.